README.md

```diff
@@ -44,15 +44,32 @@ Segmentation and depth-maps are required to use new images as background. Sample
 For an explanation of the fields in `dset.h5` (e.g.: `seg`,`area`,`label`), please check this [comment](https://github.com/ankush-me/SynthText/issues/5#issuecomment-274490044).
 
 ### Pre-processed Background Images
-The 8,000 background images used in the paper, along with their segmentation and depth masks, have been uploaded here:
-`http://thor.robots.ox.ac.uk/~vgg/data/scenetext/preproc/<filename>`, where, `<filename>` can be:
 
-|    filenames    | size |                      description                     |             md5 hash             |
-|:--------------- | ----:|:---------------------------------------------------- |:-------------------------------- |
-| `imnames.cp`    | 180K | names of images which do not contain background text |                                  |
-| `bg_img.tar.gz` | 8.9G | images (filter these using `imnames.cp`)             | 3eac26af5f731792c9d95838a23b5047 |
-| `depth.h5`      |  15G | depth maps                                           | af97f6e6c9651af4efb7b1ff12a5dc1b |
-| `seg.h5`        | 6.9G | segmentation maps                                    | 1605f6e629b2524a3902a5ea729e86b2 |
+The 8,000 background images used in the paper, along with their
+segmentation and depth masks, is included in the [same
+torrent](https://academictorrents.com/details/2dba9518166cbd141534cbf381aa3e99a087e83c)
+as the pre-generated dataset under the `bg_data` directory.  The files are:
+
+|    filenames    |                      description                     |
```

Comment:
may be include the size of the files (from the table below) here as well?

Reply:
That information is part of the torrent, same as the checksum so I didn't include it here on purpose. When one adds a torrent to download, it is provided a list of files and directories and their sizes to select.